modules/src/application/ics20_fungible_token_transfer/relay_application_logic/send_transfer.rs

```diff
@@ -42,7 +42,7 @@ where
         source_channel: msg.source_channel,
         destination_port,
         destination_channel: destination_channel.clone(),
-        data: vec![],
+        data: vec![0],
```

Comment:
Can you please explain the rationale behind this change?

Reply:
The data field should not be empty. It comes from the spec of ICS 20. Here we do not have the logic of the token transfer but data in the transfer message corresponds to ```FungibleTokenPacketData{denomination, amount, sender, receiver}``` which cannot be empty.

Reply:
I see, thanks!